.github/workflows/goalie-on-pr-review-or-response.yaml

```diff
@@ -0,0 +1,55 @@
+# on a review from someone in the reviewers group, remove the awaiting-review label and add the awaiting-author label
+name: Update labels for awaiting review
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5
+        id: get-reviewers
+        with:
+          github-token: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+          script: |
+            const {data: reviewerMembers} = await github.rest.teams.listMembersInOrg({
+              org: context.payload.organization.login,
+              team_slug: 'reviewers'
+            });
+            return reviewerMembers;
+      - uses: actions/github-script@v5
+        id: fix-labels
+        with:
+          script: |
+            const isReviewer = ${{steps.get-reviewers.outputs.result}}.some(({login}) => login === context.actor);
+            const isAuthor = context.payload.pull_request.user.login === context.actor
+             if (isAuthor) {
+              github.rest.issues.removeLabel({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                name: 'awaiting-author'
+              }).catch(() => {});
+              github.rest.issues.addLabels({
+                issue_number: context.issue.number,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                labels: ['awaiting-review']
+              });
```

Comment:
don't we need to catch the additions too? if i or the author comment twice on the same thing in a row, will this second thing explode?

Reply:
Not sure what you mean here?